docs/tasks/tools/install-kubectl.md

```diff
@@ -7,24 +7,24 @@ redirect_from:
 - "/docs/tasks/kubectl/install/"
 - "/docs/tasks/kubectl/install.html"
 ---
+{% capture overview %}
+Use the Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl), to deploy and manage applications on Kubernetes. Using kubectl, you can:
 
-To deploy and manage applications on Kubernetes, you'll use the
-Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/). It
-lets you inspect your cluster resources, create, delete, and update
-components, and much more. You will use it to look at your new cluster
-and bring up example apps.
+  * Inspect your cluster resources.
```

Comment:
Grrr. It's not rendering the bullet list correctly. I think because it's right before the ToC. Try removing the space before the asterisks or change it into a numbered list in a sentence.